docs/_static/documentation_options.js

```diff
@@ -1,6 +1,6 @@
 var DOCUMENTATION_OPTIONS = {
     URL_ROOT: document.getElementById("documentation_options").getAttribute('data-url_root'),
-    VERSION: '0.1.0',
+    VERSION: '0.1.1',
```

Comment:
Just curious, what are our rules for incrementing versions? What change constitutes a 1.0.0 vs 0.2.1 vs 0.1.2 from 0.1.1?

Reply:
Good question.  Seth might have more information as to how we are doing that with our AR apps.  I've seen the first used for breaking updates... the second for features and the third for bug fixes.  Since this PR is only bugfixes/patches, that is the reason why I've made it just an increment of the last digit.